meta-resin-common/recipes-containers/docker-disk/files/entry.sh

```diff
@@ -21,7 +22,7 @@ mkdir -p /data_disk/docker
 mkdir -p /data_disk/resin-data
 
 # Start docker with the created image.
-docker daemon -g /data_disk/docker -s btrfs &
+docker daemon -g /data_disk/docker &
```

Comment:
This runs on the yocto building machine. What if we do not have aufs in that particular kernel?